wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Field.kt

```diff
@@ -50,22 +51,25 @@ class Field private constructor(
   // Null until this field is linked.
   private var deprecated: Any? = null
 
-  // Null until this field is linked.
-  private var packed: Any? = null
-
   // Null until this field is linked.
   var isRedacted: Boolean = false
     private set
 
   val isRepeated: Boolean
     get() = label == Label.REPEATED
 
+  @Deprecated("Proto2 concept inexistant in proto3. Use EncodeMode instead.")
```

Comment:
Did you mean: **_inexistent_**

Reply:
I like to speak obsolete words
![image](https://user-images.githubusercontent.com/1767669/79047632-d802a280-7be5-11ea-8d2d-84b74aac08b4.png)